lib/ansible/modules/network/eos/_eos_template.py

```diff
@@ -181,11 +181,9 @@ def main():
                            mutually_exclusive=mutually_exclusive,
                            supports_check_mode=True)
 
```

Comment:
@bcoca The reason for me to update the deprecated files is that we don't want people to use those as a reference of how to do it. If we don't have any bad examples, people can't copy them either.